keyboards/s_ol/0xc_pad/0xc_pad.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 s-ol
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT(                                                 \
+                                                                \
+    k01, k05, k02, k06, k10, k03, k07, k11, k15, k08, k12, k16) \
+                                                                \
+    { {k01, k05, KC_NO, KC_NO}, {k02, k06, k10, KC_NO}, {k03, k07, k11, k15}, {KC_NO, k08, k12, k16}, }
```

Comment:
Admittedly, this is a nitpick, but these are supposed to resemble the assembled board.

```suggestion
#define LAYOUT(         \
         k01, k05,      \
       k02, k06, k10,   \
    k03, k07, k11, k15, \
       k08, k12, k16    \
) { \
    { k01,   k05, KC_NO, KC_NO }, \
    { k02,   k06, k10,   KC_NO }, \
    { k03,   k07, k11,   k15   }, \
    { KC_NO, k08, k12,   k16   }  \
}
```

Reply:
Ah, well good you spotted this - it used to look like that but I must've run clang-format once too many.